README.md

```diff
@@ -23,6 +24,229 @@ Terra Toolkit is a utility module used to facilitate independent development of
 
 - Install with [npm](https://www.npmjs.com): `npm install terra-toolkit --save-dev`
 
+## Webdriver.io Utility
+[Webdriver.io](http://webdriver.io/) is a framework for writing webdriver powered tests to validate functionality in browsers. Webdriver.io also provides easy services for setting up selenium, starting webpack and static servers, as well as accessibilty and visual regression testing.
+
+To make testing easier, Terar toolkit provides default configuration that enables the following:
+
+* Selenium docker instances for a consistent testing environment
+* Axe accessibility Utility
+* Visual regression testing
+* Viewport resizing helpers
+* Mocha test framework.
+
+### Installation
+1. Install docker on your machine: https://www.docker.com/
+
+2. In your root directory, create a `wdio.conf.js` file that inherits from Terra Toolkit's base config.
+
+```js
+const wdioConf = require('terra-toolkit/wdio/conf');
+const localIP = require('ip');
+
+const staticServerPort = 4567;
+
+const config = {
+  baseUrl: `http://${localIP.address()}:${staticServerPort}`,
+  // TOOD: Custom wdio config goes here. See: http://webdriver.io/guide/testrunner/configurationfile.html
+  ...wdioConf.config,
+};
+
+exports.config = config;
+```
+
+### Selenium Docker
+Selenium docker is provided as a convenience to make selenium testing easier and more stable. Running selenium in a container ensures a consistent testing environment across testing environments which is critical for visual regression testing.
+
+#### Options
+
+Under the key `seleniumDocker` in your wdio.conf.js you can pass a configuration object with the following structure:
+
+* **cidfile** - The name of the docker cidfile used to manage the docker instance during tests. Defaults to '.docker_selenium_id',
+* **enabled** - Flat to disable selenium docker; useful for CI environments which can startup the docker instance outside of test runs. Defaults to true
+* **cleanup** - Destroy the docker container after the test run. Defaults to false.
+* **image** - The docker image to use for test runs. Defaults to `selenium/standalone-chrome` or `selenium/standalone-firefox` based on browser capabilities specified in config.
+
+#### Example
+```js
+// wdio.conf.js
+const wdioConf = require('terra-toolkit/wdio/conf');
+const localIP = require('ip');
+
+const staticServerPort = 4567;
+
+const config = {
+  baseUrl: `http://${localIP.address()}:${staticServerPort}`,
```

Comment:
Is this required for correct setup?

Reply:
For this project yes.... It is critical to allow the docker container to find the host. This setup also coincidentally allows a remote selenium hub to access the host machine (assuming it can be accessed through the network).

Reply:
If this is required for correct setup, we should somehow highlight it or instruct people that baseURL must be provided. From looking at this, it sorta looks like an additional custom config variable people might assume we are overriding.

Reply:
Some of this comes down to understanding what you are consuming. The examples have baseUrl configured and it should be obvious its needed. It's not something special for toolkit

Reply:
I can add a comment in the code example